### PR TITLE
If units can have `chargeSound`, why not `chargeTurret`s too?

### PR DESCRIPTION
**Pull request by MEEPofFaith**
And I didn't forget the import this time.

### CODE REVIEW

**Comment by GlennFolker** [comment]
your code doesn't even compile.

**Review by GlennFolker — CHANGES_REQUESTED** [review]
chargeSound.at(x + tr.x, y + tr.y, Mathf.random(0.8f, 1f))

**Review by GlennFolker — CHANGES_REQUESTED** [review]
chargeSound.at(x + tr.x, y + tr.y, Mathf.random(0.8f, 1f))

**Comment by GlennFolker** [comment]
why did that sent twice

**Comment by MEEPofFaith** [comment]
I know it doesn't comepile, Im tyrig to fix it

**Comment by MEEPofFaith** [comment]
But something seems to be wrong with my Gradle because it's getting a different error message than Travis.

**Comment by MEEPofFaith** [comment]
Despite the fact that I have the same Gradle and OpenJDK version as Evl, I have an error compiling while she doesn't. I don't know why this is happening, but whatever the case I am currently unable to compile Mindustry and don't know how to fix this.

**Comment by Gdeft** [comment]
`import mindustry.gen.*;`

**Comment by MEEPofFaith** [comment]
so that's what I was missing

**Comment by Gdeft** [comment]
~~Did I just delete my comment by accident?~~

**Comment by MEEPofFaith** [comment]
Still the same issue

**Comment by GlennFolker** [comment]
¯\_(ツ)_/¯

**Comment by MEEPofFaith** [comment]
h
image

**Comment by Gdeft** [comment]
amazing

**Comment by GlennFolker** [comment]
I cant use the ones in `mindustry.annotations.Annotations.*` as well

**Comment by GlennFolker** [comment]
also @Nullable is in arc iirc

**Comment by MEEPofFaith** [comment]
So why is it broken in every file, not just the one I edited.

**Comment by GlennFolker** [comment]
> ¯\_(ツ)_/¯

**Comment by MEEPofFaith** [comment]
well, travis says it's ok, so I'll trust that. i just don't konw why i can't compile mindustyr

**Comment by GlennFolker** [comment]
let's just wait for cat's response

**Comment by Gdeft** [comment]
"no"

**Comment by RandomTNT** [comment]
Ok

File: core/src/mindustry/world/blocks/defense/turrets/ChargeTurret.java [hunk 1]
```diff
@@ -28,7 +32,8 @@ public void shoot(BulletType ammo){
 
             tr.trns(rotation, size * tilesize / 2f);
             chargeBeginEffect.at(x + tr.x, y + tr.y, rotation);
-
+            chargeSound.at(x + tr.x, y + tr.y, Mathf.random(0.8f, 1f));
```
Comment (Anuken) [inline core/src/mindustry/world/blocks/defense/turrets/ChargeTurret.java:35]: Pitch should not be random.

Comment (MEEPofFaith) [reply core/src/mindustry/world/blocks/defense/turrets/ChargeTurret.java:35]: Ok, I'll change that, though I did copy the code for the shootSound.

**Comment by MEEPofFaith** [comment]
I still don't know why I can't compile Mindustry. Travis seems to work perfectly fine by mine doesn't.

**Comment by Anuken** [comment]
> I still don't know why I can't compile Mindustry. Travis seems to work perfectly fine but mine doesn't.

You have an unncessary Arc copy, most likely.

File: core/src/mindustry/world/blocks/defense/turrets/Turret.java [hunk 2]
```diff
@@ -432,7 +432,7 @@ protected void effects(){
 
             fshootEffect.at(x + tr.x, y + tr.y, rotation);
             fsmokeEffect.at(x + tr.x, y + tr.y, rotation);
-            shootSound.at(tile, Mathf.random(0.9f, 1.1f));
+            shootSound.at(x + tr.x, y + tr.y, 1);
```
Comment (Anuken) [inline core/src/mindustry/world/blocks/defense/turrets/Turret.java:435]: The pitch for the *charge* sound shouldn't be random, but the normal sound should be.

**Comment by MEEPofFaith** [comment]
What is an unnecessary Arc copy, and how did I get one when I originally forked?

**Comment by ThePythonGuy3** [comment]
you basically got an ../Arc folder if i got it correctly

**Pull request merged**